ddtrace/appsec/processor.py

```diff
@@ -155,7 +155,11 @@ def __attrs_post_init__(self):
             try:
                 with open(self.rules, "r") as f:
                     rules = json.load(f)
+                    if config._api_security_enabled:
+                        with open(DEFAULT.API_SECURITY_PARAMETERS, "r") as f_apisec:
+                            rules.update(json.load(f_apisec))
```

Comment:
here, the `rules` field from `self.rules` is overriden by the `rules` field from the API sec file.

Reply:
Thank you. 
The code was written when there was no rules section in this file, then it was added for the tests.